frontend/src/Store/index.ts

```diff
@@ -1,16 +1,32 @@
 import { createStore, applyMiddleware } from 'redux'
 import { composeWithDevTools } from 'redux-devtools-extension'
+import createSagaMiddleware from 'redux-saga'
 import rootReducer from './rootReducer'
+import rootSaga from './rootSaga'
 
-export type RootState = ReturnType<typeof rootReducer>
+const middlewares = []
 
 const composeEnhancer = composeWithDevTools({
   // Specify name here, actionsBlacklist, actionsCreator and other options if needed
 })
 
+const sagaMiddleware = createSagaMiddleware()
+middlewares.push(sagaMiddleware)
+
 const storeEnhancer = composeEnhancer(
-  applyMiddleware(),
+  applyMiddleware(...middlewares),
   // other enhancer if any
 )
 
-export default createStore(rootReducer, storeEnhancer)
+const store = createStore(rootReducer, storeEnhancer)
+// kick off root saga
+sagaMiddleware.run(rootSaga)
+
+// const persistor = setupPersistor(store);
```

Comment:
remove unused code

Reply:
```suggestion
// const persistor = setupPersistor(store);
```
It was kept intentionally to show on how we can persist redux-store should we need it later. 

line 22 can be removed